dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -270,26 +270,27 @@ def hoc_signups
   def retention
     require 'cdo/properties'
     SeamlessDatabasePool.use_persistent_read_connection do
-      @scripts = params[:scripts_ids].present? ?
-        params[:scripts_ids].split(',').map(&:to_i) :
-        [1, 17, 18, 19, 23]  # Default to the CSF scripts.
-      # Get the cached retention_stats from the DB, trimming those stats to only the requested
+      @all_scripts_names_ids = Script.order(:id).pluck(:name, :id).to_h
```

Comment:
Note that we need to grab all the scripts to populate the list of choices...

Depending on use cases, it may or may not make sense to add where(hidden: false) to shorten the list of @all_scripts_names_ids.